diff --git a/node_modules/react-native-easy-toast/index.js b/node_modules/react-native-easy-toast/index.js
index d30971e..488d522 100755
--- a/node_modules/react-native-easy-toast/index.js
+++ b/node_modules/react-native-easy-toast/index.js
@@ -13,10 +13,11 @@ import {
     Animated,
     Dimensions,
     Text,
-    TouchableWithoutFeedback,
-    ViewPropTypes as RNViewPropTypes,
+    TouchableWithoutFeedback
 } from 'react-native'
 
+import { ViewPropTypes as RNViewPropTypes } from 'deprecated-react-native-prop-types';
+
 import PropTypes from 'prop-types';
 const ViewPropTypes = RNViewPropTypes || View.propTypes;
 export const DURATION = {
@@ -153,7 +154,7 @@ Toast.propTypes = {
         'center',
         'bottom',
     ]),
-    textStyle: Text.propTypes.style,
+    textStyle: PropTypes.any,
     positionValue:PropTypes.number,
     fadeInDuration:PropTypes.number,
     fadeOutDuration:PropTypes.number,
